src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -140,6 +149,13 @@ public AddressBook load() throws StorageOperationException {
             throw new StorageOperationException("File contains illegal data values; data type constraints not met");
         }
     }
+    
+    public void checkFileNotMissing() throws FileMissingException {
+    	File file = new File(this.getPath());
+    	if(!file.exists()) {
+    		throw new FileMissingException("Storage file is missing.");
+    	}
+    }
```

Comment:
Throwing an exception from this method seems like an overkill. A `boolean isStorageFilePresent()` is more suitable?